start

```diff
@@ -75,6 +75,7 @@ docker run -d --rm \
 -e TASK_QUEUE_SLOTS=1 \
 -e PRIORITY_TASK_QUEUE_SLOTS=1 \
 -e INFERENCE_DIR=$INFERENCE_DIR \
+-e SECRET_KEY=default \
```

Comment:
so we don't forget about it let's add a note to the `https://github.com/code-kern-ai/deployment-managed-app/pull/12` that we need to add the env variable

Reply:
yes right I will do this

Reply:
now the secret key for managed is also default. I think it should be different for our applications

Reply:
Set to variable and set on managed server a secret key in env file